spyglass/src/androidTest/java/com/linkedin/android/spyglass/ui/MentionsEditTextTest.java

```diff
@@ -97,4 +97,21 @@ private void testMention(String hello, Mentionable mention) throws Exception {
         // ensure mention does not clobber existing text
         assertTrue(mEditText.getText().toString().startsWith("Hello "));
     }
+
+    @Test
+    public void testInsertMentionWithoutToken() throws Exception {
+        Mentionable mention = new TestMention("FirstName MiddleName LastName");
+        mEditText.insertMentionWithoutToken(mention);
+
+        // ensure mention adds correctly
+        assertTrue(mEditText.getText().toString().startsWith("FirstName MiddleName LastName"));
```

Comment:
Is there any particular reason why you're using "startsWith" and not "equals" here?